src/main/java/uk/ac/ebi/embl/converter/fftogff3/GFF3AnnotationFactory.java

```diff
@@ -129,22 +129,30 @@ private GFF3Feature transformFeature(String accession, Feature ffFeature, Option
                     q -> q.isValue() ? q.getValue() : "true", // Ensure non-empty values
                     (existing, replacement) -> existing));
 
-    gene.ifPresent(v -> attributes.put("gene", v));
+    gene.ifPresent(v -> base_attributes.put("gene", v));
+    List<GFF3Feature> features = new ArrayList<>();
```

Comment:
This declaration can be moved to the beginning of the method and rename member to 
`List<GFF3Feature> gff3Features = new ArrayList<>();`